internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/fastclient/FastClientIndividualFeatureConfigurationTest.java

```diff
@@ -72,9 +73,11 @@ public void testServerReadQuota() throws Exception {
     }
     String readQuotaStorageNodeTokenBucketRemaining =
         ".venice-storage-node-token-bucket--QuotaRcuTokensRemaining.Gauge";
-    String readQuotaRequestedString = "." + storeName + "--quota_rcu_requested.Count";
-    String readQuotaRejectedString = "." + storeName + "--quota_rcu_rejected.Count";
-    String readQuotaAllowedUnintentionally = "." + storeName + "--quota_rcu_allowed_unintentionally.Count";
+    String readQuotaRequestedQPSString = "." + storeName + "--quota_request.Rate";
+    String readQuotaRejectedQPSString = "." + storeName + "--quota_rejected_request.Rate";
+    String readQuotaRequestedKPSString = "." + storeName + "--quota_request_key_count.Rate";
+    String readQuotaRejectedKPSString = "." + storeName + "--quota_rejected_key_count.Rate";
+    String readQuotaAllowedUnintentionally = "." + storeName + "--quota_unintentionally_allowed_key_count.Count";
```

Comment:
for my curiosity, why do we have quota assigned to clients unintentionally?

Reply:
SN quota right now is fail pass. Meaning if for some reason SN didn't initialize properly or we can't fetch/calculate the quota then the enforcer will pretend the user has infinite quota and allow the requests unintentionally. We can change this behavior later once we believe the quota system is stable enough to do so (this metric never went above zero since we fixed some delayed CV update after restart which was months ago).